packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/dashboard/manage-nsx-edges/dedicatedCloud-datacenter-manage-nsx.module.js

```diff
@@ -7,6 +7,10 @@ const moduleName = 'ovhManagerDedicatedCloudDatacenterManageNsxEdges';
 
 angular
   .module(moduleName, [datacenterManageNsxEdgesComponent])
+  .constant(
+    'TRACKING_PREFIX',
```

Comment:
the name is too generic and can be override by error in any another module
```suggestion
    'NSX_TRACKING_PREFIX',
```